BuildResidentialHPXML/measure.rb

```diff
@@ -1125,11 +1125,23 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('Btu/hr')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_heating_airflow', false)
+    arg.setDisplayName('Heating System: Heating Airflow')
```

Comment:
We only want to expose airflow rate inputs for central systems (air distribution). I think you can replace all the airflow arguments with just two arguments (`hvac_distribution_heating_airflow` and `hvac_distribution_cooling_airflow`).

You'll also need to update the schematron, documentation, etc. to remove the airflow inputs from non-central systems (like wall furnace, etc.).

Reply:
Why only for central systems, e.g., not floor/wall furnace? These systems output HeatingAirflowCFM...

Reply:
You're right that we do, but it's hard for me to imagine a BuildResHPXML user ever wanting to change the airflow for a room AC of wall furnace, the input will have almost no effect. It feels like overkill to introduce so many arguments when two will do.